content/bn/docs/reference/glossary/annotation.md

```diff
@@ -0,0 +1,16 @@
+---
+title: এনোটেশন
+id: annotation
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/annotations
```

Comment:
/bn

Reply:
Does that page exist in the `dev-1.24-bn.1` branch?